example/GraphClientSimpleExample.py

```diff
@@ -28,38 +26,72 @@
-        print(json.dumps(json_obj, indent=2, sort_keys=True))
-
-        client.execute(
-            'CREATE SPACE IF NOT EXISTS test(vid_type=FIXED_STRING(30)); USE test;'
+        # prepare space and insert data
+        print("\n Prepare space data...\n")
+        resp = client.execute(
+            'CREATE SPACE IF NOT EXISTS test(vid_type=FIXED_STRING(30));'
+        )
+        assert resp.is_succeeded(), resp.error_msg()
+        time.sleep(6)
+        resp = client.execute('USE test;')
+        assert resp.is_succeeded(), resp.error_msg()
+        resp = client.execute(
             'CREATE TAG IF NOT EXISTS person(name string, age int);'
             'CREATE EDGE like (likeness double);'
         )
 
+        time.sleep(6)
         # insert data need to sleep after create schema
+        resp = client.execute('CREATE TAG INDEX person_age_index on person(age)')
         time.sleep(6)
-
         # insert vertex
         resp = client.execute(
             'INSERT VERTEX person(name, age) VALUES "Bob":("Bob", 10), "Lily":("Lily", 9)'
         )
         assert resp.is_succeeded(), resp.error_msg()
-
         # insert edges
         resp = client.execute('INSERT EDGE like(likeness) VALUES "Bob"->"Lily":(80.0);')
         assert resp.is_succeeded(), resp.error_msg()
+        resp = client.execute('REBUILD TAG INDEX person_age_index')
+        assert resp.is_succeeded(), resp.error_msg()
+        print("\n Data preparation is completed\n")
 
+        # test fetch prop on statement
+        print("\n Test fetch prop on statement...\n")
         resp = client.execute('FETCH PROP ON person "Bob" YIELD vertex as node')
         assert resp.is_succeeded(), resp.error_msg()
         print_resp(resp)
-
         resp = client.execute('FETCH PROP ON like "Bob"->"Lily" YIELD edge as e')
         assert resp.is_succeeded(), resp.error_msg()
         print_resp(resp)
 
+        # get the result in json format
+        print("\n Test execute_json interface...\n")
+        resp_json = client.execute_json("yield 1")
+        json_obj = json.loads(resp_json)
+        print(json.dumps(json_obj, indent=2, sort_keys=True))
+
+        # test parameter interface
+        print("\n Test cypher parameter...\n")
+        bval = ttypes.Value()
+        bval.set_bVal(True)
+        ival = ttypes.Value()
+        ival.set_iVal(3)
+        sval = ttypes.Value()
+        sval.set_sVal("Cypher Parameter")
+        params = {"p1": ival, "p2": bval, "p3": sval}
+        resp = client.execute_parameter(
+            'RETURN abs($p1)+3, toBoolean($p2) and false, toLower($p3)+1', params
+        )
+        assert resp.is_succeeded(), resp.error_msg()
+        print_resp(resp)
+        resp = client.execute_parameter(
+            'MATCH (v:person)--() WHERE v.age>abs($p1)+3 RETURN v,v.age AS vage ORDER BY vage, $p3 LIMIT $p1+1',
+            params,
+        )
+        assert resp.is_succeeded(), resp.error_msg()
+        print_resp(resp)
+
```

Comment:
This part could be put to the test. Let's keep the example concise.

Reply:
ACK.